runtime/doc/eval.txt

```diff
@@ -7271,6 +7272,23 @@ stdioopen({opts})			 *stdioopen()*
 		  - 0 on invalid arguments
 
 
+stdpath({what})					*stdpath()* *E6100*
+		Returns the standard path(s) for {what}.
+
+		These directories are the default locations for various files
+		used by Neovim.
+
+		{what}       Type    Description ~
+		cache        String  Cache directory. Useful for plugins
+		                     that need temporary files to work.
+		config       String  User configuration directory. The
+		                     |init.vim| is stored here.
+		config_dirs  List    Additional configuration directories.
+		data         String  User data directory. |shada-file|
+		                     are stored here for example.
```

Comment:
Should read:

> The |shada-file| is stored here.

like the `init.vim` example above. If there is a reason to keep "for example", it needs a comma before it (otherwise it implies the reason the ShaDa file resides there is to serve as an example).

Reply:
Thanks.  I made that change.